packages/js/e2e-core-tests/specs/merchant/wp-admin-page-loads.test.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Internal dependencies
+ */
+const { merchant } = require( '@woocommerce/e2e-utils' );
+const { MENUS } = require( '../data/elements' );
+
+/**
+ * External dependencies
+ */
+const { it, describe, beforeAll } = require( '@jest/globals' );
+
+const runPageLoadTest = () => {
+	describe.each( MENUS )(
+		' %s > Opening Top Level Pages',
+		( menuTitle, menuElement, subMenus ) => {
+			beforeAll( async () => {
+				await merchant.login();
+				await page.setViewport( {
```

Comment:
@Todo

Remove setting the viewport when this [issue](https://github.com/woocommerce/woocommerce/issues/31305) is fixed.